_config.yml

```diff
@@ -1,21 +1,121 @@
-title: Konveyor Move2Kube
-description: Accelerate the process of re-platforming to Kubernetes/Openshift.
-show_downloads: true
-google_analytics:
-theme: jekyll-theme-cayman 
-baseurl: /move2kube
-home_url: https://konveyor.github.io/move2kube/
-github_url: http://github.com/konveyor/move2kube
-slack_url: https://kubernetes.slack.com/archives/CR85S82A2
-forum_url: https://groups.google.com/forum/#!forum/konveyorio
-installation_url: /installation
-getting_started_url: /getting-started
-tutorials_url: /tutorial
-gems:
-- jekyll-seo-tag
+# Welcome to Jekyll!
+#
+# This config file is meant for settings that affect your whole site, values
+# which you are expected to set up once and rarely edit after that. If you find
+# yourself editing these this file very often, consider using Jekyll's data files
+# feature for the data you need to update frequently.
+#
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'jekyll serve'. If you change this file, please restart the server process.
+
+# Site settings
+# These are used to personalize your new site. If you look in the HTML files,
+# you will see them accessed via {{ site.title }}, {{ site.github_repo }}, and so on.
+# You can create any custom variable you would like, and they will be accessible
+# in the templates via {{ site.myvariable }}.
+title: Konveyor MoveKube
+description: A Jekyll theme for documentation
```

Comment:
Can you change the description to Move2Kube description?

Reply:
Yes sure, I have changed it. It was there on my to-do-list but I missed correcting it.